spec/policies/event_policy_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe EventPolicy do
+
+  let(:reviewer) { create(:user, :reviewer) }
+  let(:organizer) { create(:organizer) }
+  let(:admin) { create(:admin) }
+  let(:event) { create(:event) }
+
+  subject { described_class }
+
+  #permissions ".scope" do
+  #  pending "add some examples to (or delete) #{__FILE__}"
+  #end
+
+  permissions :show?, :create?, :update?, :destroy? do
```

Comment:
We'll want to break these up since permissions are more subtle than this.  I'll make role specific comments down below.